staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -538,8 +540,13 @@ See http://k8s.io/docs/reference/using-api/api-concepts/#conflicts`, err)
 		return err
 	}
 
+	metadata, _ = meta.Accessor(info.Object)
```

Comment:
I'm assuming you moved the "Accessor" above (L411) so that you would be able to access it here? so I don't think that is needed, is it?